src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -160,6 +160,12 @@ private function build_event_array( array $record ) {
 			$event['tax_input'][ Tribe__Events__Main::TAXONOMY ] = $this->translate_terms_to_ids( explode( ',', $cats ) );
 		}
 
+		$additionalFields = tribe_get_option( 'custom-fields' );
+		foreach ( $additionalFields as $field ) {
+			$key = $field['name'];
+			$event[$key] = $this->get_value_by_key( $record, $key );
```

Comment:
Check this on all the changes you applied, we normally use the following structure for variable `array` index:
```
$event[ $key ] = ...
```